bioblend/galaxy/notifications/__init__.py

```diff
@@ -0,0 +1,577 @@
+"""
+Contains possible interaction dealing with Galaxy notifications.
+"""
+
+from datetime import datetime
+from typing import (
+    Any,
+    Dict,
+    List,
```

Comment:
`Literal` is part of Python 3.8 (oldest release supported by BioBlend).

```suggestion
    List,
    Literal,
```